Lusas_Engine/Create/LusasConfig.cs

```diff
@@ -30,12 +30,15 @@ public static partial class Create
         /****           Public Constructors             ****/
         /***************************************************/
 
-        public static LusasConfig LusasConfig(LibrarySettings librarySettings = null)
+        public static LusasConfig LusasConfig(LibrarySettings librarySettings = null, double mergeTolerance = double.NaN)
```

Comment:
```suggestion
        public static LusasConfig LusasConfig(LibrarySettings librarySettings = null, double mergeTolerance = Tolerance.Distance)
```

Will need to add `Geometry_oM` reference.